js/templates/modals/settings/page.html

```diff
@@ -79,10 +79,11 @@ <h2 class="h3 txUnb"><%= ob.polyT('settings.pageTab') %></h2>
     <div class="flexRow">
       <div class="col3">
         <label><%= ob.polyT('settings.avatar') %></label>
+        <div class="clrT2 txBase"><%= ob.polyT('settings.loadAvatarHelp', {minWidth: ob.avatarMinWidth, minHeight: ob.avatarMinHeight}) %></div>
       </div>
       <div class="col9">
         <div class="flexRow gutterH" id="avatarCropper">
-          <div class="contentBox avatarPreview clrP clrBr flexNoShrink js-avatarPreview"></div>
+          <div class="contentBox avatarPreview clrP flexNoShrink js-avatarPreview"></div>
```

Comment:
Good point. However, when I leave the border in there, it results in this:

![screenshot 2016-12-16 15 01 55](https://cloud.githubusercontent.com/assets/13066886/21281328/a2e095fe-c3a0-11e6-8271-46501ca7a64d.png)

I can't figure out where to put the border color so that it wraps the circle instead of the container...